### ISSUE TITLE
examples throw error

### ISSUE THREAD

**Issue by despairblue** [vigetlabs/microcosm#60]
all examples throw
```
invariant.js?96aa*:42 Uncaught Error: Invariant Violation: _registerComponent(...): Target container is not a DOM element.
invariant	@	invariant.js?96aa*:42
ReactMount._registerComponent	@	ReactMount.js?2450*:355
ReactMount._renderNewRootComponent	@	ReactMount.js?2450*:395
ReactPerf.measure.wrapper	@	ReactPerf.js?6da6*:70
ReactMount.render	@	ReactMount.js?2450*:493
ReactPerf.measure.wrapper	@	ReactPerf.js?6da6*:70
(anonymous function)	@	index.jsx:35
(anonymous function)	@	Microcosm.js?0648*:258
(anonymous function)	@	Microcosm.js?0648*:257
install	@	install.js?d8f2*:8
start	@	Microcosm.js?0648*:256
exports.default.obj.__esModule.default	@	index.jsx:29
__webpack_require__	@	bootstrap a628ef5f159c41ff2674?2f37*:19
(anonymous function)	@	bootstrap a628ef5f159c41ff2674?2f37*:39
(anonymous function)	@	bootstrap a628ef5f159c41ff2674?2f37*:39
```

**Comment by nhunzaker** [comment]
Yikes. Looks like this is with how I have webpack dev server configured. I"ll take a look.

**Comment by nhunzaker** [comment]
Looks like this happened because webpack was serving its own index.html for the examples unless you specifically visited `/example/index.html`.

I've made a dedicated examples page. It should alleviate the issue, but let me know if things still aren't working for you.

**Comment by despairblue** [comment]
:clap: works now, except that the advance example throws an error:

```
Warning: Failed propType: Required prop `lists` was not specified in `Body`. Check the render method of `Home`.
warning.js:48 Warning: Failed propType: Required prop `items` was not specified in `Body`. Check the render method of `Home`.
Body.jsx:50 Uncaught TypeError: Cannot read property 'map' of undefinedrender @ Body.jsx:50
ReactCompositeComponentMixin._renderValidatedComponentWithoutOwnerOrContext @ ReactCompositeComponent.js:789
ReactCompositeComponentMixin._renderValidatedComponent @ ReactCompositeComponent.js:816
ReactPerf.measure.wrapper @ ReactPerf.js:70
ReactCompositeComponentMixin.mountComponent @ ReactCompositeComponent.js:237
ReactPerf.measure.wrapper @ ReactPerf.js:70
ReactReconciler.mountComponent @ ReactReconciler.js:38
ReactMultiChild.Mixin.mountChildren @ ReactMultiChild.js:192
ReactDOMComponent.Mixin._createContentMarkup @ ReactDOMComponent.js:289
ReactDOMComponent.Mixin.mountComponent @ ReactDOMComponent.js:199
ReactReconciler.mountComponent @ ReactReconciler.js:38
ReactCompositeComponentMixin.mountComponent @ ReactCompositeComponent.js:247
ReactPerf.measure.wrapper @ ReactPerf.js:70
ReactReconciler.mountComponent @ ReactReconciler.js:38
ReactCompositeComponentMixin.mountComponent @ ReactCompositeComponent.js:247
ReactPerf.measure.wrapper @ ReactPerf.js:70
ReactReconciler.mountComponent @ ReactReconciler.js:38
ReactCompositeComponentMixin.mountComponent @ ReactCompositeComponent.js:247
ReactPerf.measure.wrapper @ ReactPerf.js:70
ReactReconciler.mountComponent @ ReactReconciler.js:38
ReactCompositeComponentMixin.mountComponent @ ReactCompositeComponent.js:247
ReactPerf.measure.wrapper @ ReactPerf.js:70
ReactReconciler.mountComponent @ ReactReconciler.js:38
mountComponentIntoNode @ ReactMount.js:248
Mixin.perform @ Transaction.js:134
batchedMountComponentIntoNode @ ReactMount.js:269
Mixin.perform @ Transaction.js:134
ReactDefaultBatchingStrategy.batchedUpdates @ ReactDefaultBatchingStrategy.js:66
batchedUpdates @ ReactUpdates.js:110
ReactMount._renderNewRootComponent @ ReactMount.js:404
ReactPerf.measure.wrapper @ ReactPerf.js:70
ReactMount.render @ ReactMount.js:493
ReactPerf.measure.wrapper @ ReactPerf.js:70
register @ render.jsx:32
__start @ plugin.js:14
install @ install.js:14
(anonymous function) @ install.js:15
register @ router.js:46
__start @ plugin.js:14
install @ install.js:14
(anonymous function) @ install.js:15
register @ storage.js:30
__start @ plugin.js:14
install @ install.js:14
start @ Microcosm.js:256
exports.default.obj.__esModule.default @ index.jsx:41
__webpack_require__ @ bootstrap c61248dfd95f5c369219:19
(anonymous function) @ bootstrap c61248dfd95f5c369219:39
(anonymous function) @ bootstrap c61248dfd95f5c369219:39
```

By the way, do you think microcosm could be used for a react-native app?

**Comment by nhunzaker** [comment]
Hmmm. No dice on my end. Have you opened the examples before? We cache data in localStorage for that example, what happens if you clear it?

**Comment by nhunzaker** [comment]
I never followed up on this... I apologize. The examples should work, and Microcosm should now work in React Native. It did for a long time, however in the past few releases I added added a debug warning long the lines of:

```
You may have imported an action from the wrong namespace, like:
import { badNamespace } from 'actions'
```

Even though it was a string, it through off the ReactNative JS parser. I've fixed this. I'll work with the ReactNative project to see if we can fix the parse issue as well.
